changelog/fragments/update-kb-from-36124ae2e027-to-9c02d557f01b.yaml

```diff
@@ -0,0 +1,19 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      (go/v2) Added the command `òperator-sdk edit` which allows users edit the project layout to support Multi-Group.
+      More info: https://book.kubebuilder.io/migration/multi-group.html
```

Comment:
```suggestion
      (go/v2) Added the command `òperator-sdk edit` which allows users edit the project layout to support [multi-group](https://book.kubebuilder.io/migration/multi-group.html).
```

Reply:
Also Mulit-Group really should be lower-cased, as it is not a proper noun in this context.